Frontend/src/app/assets/asset-info-dialog/asset-info-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h2 mat-dialog-title>Assets</h2>
```

Comment:
For all new files, add the Copyright header.

Reply:
Done